### PR TITLE
sept 19 blogpost

### PR DESCRIPTION
**Pull request by d-m-u**

### CODE REVIEW

**Comment by JPrause** [comment]
@juliancheal @jntullo @syncrou @Fryguy here is Drew's blog entry for this week.
Can you all review and make any comments for suggested changes.

File: source/blog/2016-09-19-last-week-in-manageiq-absence-atomic-providers.html.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Last Week in ManageIQ: An Afresh Absence of Atomic Providers"
+date: 2016-09-19
+author: drew
+comments: true
+published: true
+---
+
+Thanks for tuning in today for the topical Last Week in ManageIQ! Drew Uhlmann here with you to limn marked mint MIQ metamorphoses.
```
Comment (juliancheal) [inline source/blog/2016-09-19-last-week-in-manageiq-absence-atomic-providers.html.md]: Great intro :)

**Comment by juliancheal** [comment]
LGTM 👍

File: source/blog/2016-09-19-last-week-in-manageiq-absence-atomic-providers.html.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+published: true
+---
+
+Thanks for tuning in today for the topical Last Week in ManageIQ! Drew Uhlmann here with you to limn marked mint MIQ metamorphoses. 
+
+## Analysis
+In the past week [115 PRs][PRs merged last week] with [347 commits][Commits merged last week] were merged to master by 49 people (which could probably include you too! Simply peruse our [list of issues](https://github.com/ManageIQ/manageiq/issues), pick a particular, and produce a PR.)
+
+## Augmented
+A chic change was made to rails bin/setup file, comprising of a command to clean logs/tmp.
```
Comment (jntullo) [inline source/blog/2016-09-19-last-week-in-manageiq-absence-atomic-providers.html.md]: I like the use of "chiq" 😎

**Comment by syncrou** [comment]
@d-m-u - I love all the `a's`! - Really like the language.  Had a riot reading it!  

One small thought - it would be great to have a link to the discussed PRs in each section.

**Comment by JPrause** [comment]
@Fryguy or @chessbyte this PR for the blog entry has been reviewed and approved. Please merge.

**Comment by chessbyte** [comment]
@d-m-u Unusual writing.  Had to look up definition of `limn` just to get going!  Thanks!

**Pull request merged**